ff/src/fields/models/fp12_2over3over2.rs

```diff
@@ -230,6 +206,95 @@ pub fn characteristic_square_mod_6_is_one(characteristic: &[u64]) -> bool {
     (char_mod_6 * char_mod_6) % 6 == 1
 }
 
+impl<P: Fp12Config> CyclotomicField for Fp12<P> {
+    const INVERSE_IS_FAST: bool = true;
+
+    fn cyclotomic_inverse_in_place(&mut self) -> Option<&mut Self> {
+        self.is_zero().not().then(|| {
+            self.conjugate();
+            self
+        })
+    }
+
+    fn cyclotomic_square_in_place(&mut self) -> &mut Self {
```

Comment:
The method is now duplicated in `impl Fp12` and `impl CyclotomicField for Fp12` - it should rather be unified no?

Reply:
Fixed

Reply:
Fixed!